zdm/survey.py

```diff
@@ -591,12 +881,37 @@ def process_survey_file(self,filename:str,
         
         print("FRB survey sucessfully initialised with ",self.NFRB," FRBs starting from", self.iFRB)
 
+    
     def fix_coordinates(self,verbose=False):
         """
         Takes and FRB, and fills out missing coordinate values
         Note that now, RA, DEC, Gl, and Gb will be present
         But their default values are None
         """
+        # converts to float if in string. Will do nothing if None
```

Comment:
Am trying to intuit why this would be needed..

Reply:
Basically, if people publish an FRB in RA/DEC, or in Gl Gb, but not the other. Makes it easier for entering data. You need Gl, Gb for NE2001 estimates, and to place cuts on Gl, Gb to remove fluctuations due to the Galactic Plane